cs.py

```diff
@@ -6,11 +6,12 @@
 from modules import logger
 import rm
 import subprocess
-         
+
+
 def main():
     """ main function """
     parser = argparse.ArgumentParser(description='this is to get IP address for lynis audit only')
-    parser.add_argument('-env', '--environment', required=True, help='The cloud on which the test-suite is to be run',
```

Comment:
With env variable as a not mandatory, cs.py will fail in many scenario's
Any particular scenario, why it is required to be false?

Reply:
@jayeshchauhan